examples/java/pom.xml

```diff
@@ -10,7 +10,7 @@
     <name>Hello World REST Service</name>
 
     <properties>
-        <spring.version>3.1.2</spring.version>
+        <spring.version>3.3.4</spring.version>
```

Comment:
Do we have to callout that this only works with the most recent Spring Boot versions?

Reply:
yes - [added to](https://github.com/grafana/app-o11y/blame/0c93f905edd5b8a86b354dc724c135388cd7f0c6/docs/sources/collector/collecting-logs-through-k8s-stdout.md#L67)